lib/elixir/lib/option_parser.ex

```diff
@@ -492,4 +556,34 @@ defmodule OptionParser do
       :unknown
     end
   end
+
+  defp format_errors(errors, opts) do
+    details = Enum.map_join(errors, ". ", &format_error(&1, opts))
+    total = Enum.count(errors)
+    error_pluralize = pluralize(total, "error", "errors")
+    "#{total} #{error_pluralize} found! #{details}"
+  end
+
+  defp format_error({option, nil}, _) do
+    "Unknown option #{option}"
+  end
+
+  defp format_error({option, value}, opts) do
+    option_key =
+      String.lstrip(option, ?-)
+      |> String.to_atom()
```

Comment:
We can have this on one line maybe?

```elixir
option_key = option |> String.lstrip(?-) |> String.to_atom()
```

Feels short enough and this way we bring it down from 3 to 1 line :) Wdyt?